### ISSUE TITLE
TCK framework is confusing

### ISSUE THREAD

**Issue by zhaohaifei** [vesoft-inc/nebula#3638]
**Introduction**
The TCK framework is confusing:
  1. The entire framework is divided into test and tck, but a lot of content in test can be implemented using tck.
  2. Some methods are not abstract enough, such as NebulaService.start(), in which not only starts the nebula service, but also executes "add hosts". 
  3. Many global variables are distributed in various files, with different names, but the content is the same. For example, CURR_PATH in nebula-test-run.py, CURR_DIR and TEST_DIR in makefile.
  4. Some variables are confusing, such as NB_TMP_PATH and SPACE_TMP_PATH, which seems directory names, but are actually file names.

In short, TCK is not good enough. It's better to make some changes.

**Contents**

**Related work**

**Comment by HarrisChu** [comment]
5. too many steps in a file, we could separate the steps, e.g.
define pytest_plugins in `conftest.py`, and then separate the steps
![image](https://user-images.githubusercontent.com/1726587/148178489-5d6e62ae-1f0e-4776-974a-4c7a0c9e33d4.png)

![image](https://user-images.githubusercontent.com/1726587/148178514-f30672ab-8503-4c82-a536-edfc879d5934.png)

**Comment by Sophie-Xie** [comment]
Will be fixed in the future.
